model-engine/model_engine_server/domain/use_cases/llm_model_endpoint_use_cases.py

```diff
@@ -695,6 +733,70 @@ async def create_lightllm_bundle(
             )
         ).model_bundle_id
 
+    async def create_tensorrt_llm_bundle(
+        self,
+        user: User,
+        framework_image_tag: str,
+        endpoint_unique_name: str,
+        num_shards: int,
+        checkpoint_path: Optional[str],
+    ):
+        command = []
+
+        subcommands = []
+        if checkpoint_path is not None:
+            if checkpoint_path.startswith("s3://"):
+                subcommands += self.load_model_files_sub_commands_trt_llm(
+                    checkpoint_path,
+                )
+            else:
+                raise ObjectHasInvalidValueException(
```

Comment:
Given that the `if` is explicitly checking if it's an s3 path, maybe this error can be more specific and tell the user that only s3 paths are supported? A more generic "Not able to load checkpoint path" error can still be useful as a stopgap catch-all for `load_model_files_sub_commands_trt_llm`, until we can get more precise there too.